pyanaconda/packaging/yumpayload.py

```diff
@@ -776,7 +776,12 @@ def _getAddons(self, baseurl, proxy_url, sslverify):
             if not c.has_section(addonSection) or c.get(addonSection, "type") != "addon" or not c.has_option(addonSection, "repository"):
                 continue
 
-            url = "%s/%s" % (baseurl, c.get(addonSection, "repository"))
+            repo_location = c.get(addonSection, "repository")
+            if repo_location.startswith(('https://', 'http://', 'ftp://')):
+                log.debug("Addon %s uses full url: %s", addon, repo_location)
+                url = repo_location
+            else:
+                url = "%s/%s" % (baseurl, repo_location))
```

Comment:
Hi, could you please fix this line.
You have here one missing or one extra bracket.